app/views/lecturers/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<p id="notice"><%= notice %></p>
+
+<h1>
+  <%= @lecturer.name %>
+</h1>
```

Comment:
We dont need views for lecturers. the info for lecturers will be presented with the associated resource, subjects